packages/native/src/components/Text/index.tsx

```diff
@@ -30,6 +45,8 @@ export interface BaseTextProps
   color?: string;
   lineHeight?: number;
   bracket?: boolean;
+  textTransform?: string;
```

Comment:
There might be a reason that you used this, but if you remove it, it will use react-native TextProps type, with a nice enum